node_modules/oae-authz/tests/test-groups.js

```diff
@@ -15,10 +15,12 @@
 
 var _ = require('underscore');
 var assert = require('assert');
+var shortid = require('shortid');
```

Comment:
> I'm not sure. I think we started using PascalCasing all variables requiring modules of our own (e.g., PrincipalsDAO, ContentAPI, ..) but kept short single word variables such as fs, assert, util, ... lowercase?

I think this is what we did as well. Can we settle on it? I'll rename all `ShortId`s to `shortid`, it's not too much.